project/data_analysis/python/get_spectra_from_alms.py

```diff
@@ -89,10 +89,14 @@
     
     xtra_pw1, xtra_pw2, mm_tf1, mm_tf2 = None, None, None, None
     if deconvolve_pixwin:
-        # some xtra pixel window in the case of healpix
-        # or Planck projected into CAR
-        xtra_pw1 = transfer_function.healpix_pixwin(sv1, templates[sv1], binning_file, lmax)
-        xtra_pw2 = transfer_function.healpix_pixwin(sv2, templates[sv2], binning_file, lmax)
+        if d[f"pixwin_{sv1}"]["pix"] == "HEALPIX":
```

Comment:
what's going on if someone asks for pixwin deconvolution but set `pix` value to CAR ? There are no default value for `xtra_pw{1,2}` parameter (in the `get_alms.py` file, `inv_pixwin_lxly` is `None` by default)

Reply:
you have the line:     xtra_pw1, xtra_pw2, mm_tf1, mm_tf2 = None, None, None, None
then if you pass None to 

 lb, ps = transfer_function.deconvolve_xtra_tf(lb,
                                                                            ps,
                                                                          spectra,
                                                                         xtra_pw1=xtra_pw1,
                                                                        xtra_pw2=xtra_pw2,
                                                                         mm_tf1=mm_tf1,
                                                                   mm_tf2=mm_tf2)

nothing happen. the CAR pixwin deconvolution happen in get_alm, where it's done in map space using a FFT.